backend/src/main/twirl/MainZIOhttpZIO.scala.txt

```diff
@@ -26,14 +24,17 @@ object Main extends ZIOAppDefault {
         .options
     val app: HttpApp[Any, Throwable] = ZioHttpInterpreter(serverOptions).toHttp(Endpoints.all)
 
-    val port = sys.env.get("http.port").map(_.toInt).getOrElse(8080)
+    val port = sys.env.get("HTTP_PORT").flatMap(_.toIntOption).getOrElse(8080)
```

Comment:
nit: change from `http.port` to `HTTP_PORT` is not mentioned in the commit message... what is more it is applied only to subset of templates :/

Reply:
Other templates were changed in another commit. I will clean that up.

Reply:
Moved to another PR